script/Ciel/Day 13/13_01_02_CIEL13_7F.txt

```diff
@@ -764,7 +878,9 @@ Senpai silently looks at me like she's about to cry. After looking closer, I see
 [sha:600e1e7eafee843356ebc8fc277e27868d4bf496]{
 -- Page 464, Offset 36600.
 -- いつもの制服も法衣姿もいいけど、ここぞという時に特別な服装で決めてくるとか反則です。あんなの誰だって惚れ直します」
-Your usual school uniform and habit are good too, but busting out your special uniform at such a critical moment is definitely against the rules. Anyone would fall in love with you after seeing that."
+Don't get me wrong, I love your usual school uniform and the habit,
+ but whipping out a whole new special uniform at the eleventh hour practically counts as foul play.
+ Anyone seeing you in that would fall in love all over again."
```

Comment:
I think it's the concept of the power up stuff that's cool, not the uniform per se.

>Anyone would fall in love with you all over again with %{i}that%{/i}.